docs/architecture/adr-102-rpc-companion.md

```diff
@@ -40,109 +40,128 @@ TBD
 ### Requirements
 
 The target audience for this solution are operators and integrators that want to alleviate the load on their nodes by offloading
-the queryable data requests to the **Standalone RPC**.
+the queryable data requests to the **RPC Companion**.
 
 This solution shall meet the following requirements in order to provide real benefits to these users.
 
-The **Standalone RPC** solution shall:
+The **RPC Companion** solution shall:
 
-1. Provide an **ingestion service** implemented as a data companion that can pull data from the node and store it on
+1. Provide an **[Ingest Service](#ingest-service)** implemented as a data companion that can pull data from the node and store it on
 its own storage (database)
-2. Provide its own storage (database) that can handle a high-load of reads and also have a good performance on inserting data.
-3. Implement a schema that needs to be backwards compatible with the current CometBFT RPC.
+2. Provide its own storage ([Database](#database)) that can handle a high-load of reads and also have a good performance on inserting data.
```

Comment:
The idea here from a database perspective is that the ratio of read/writes is much higher, since the ingest will insert only a every 5 or so seconds but if load balanced so many reads.

Reply:
Ok. My point is that "good performance" is too abstract and claiming you met the requirement in the end will be subjective.

Reply:
makes sense, let me rephrase that

Reply:
I removed the blurb about performance, it was very subjective and hard to justify through more concrete numbers